plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/OAuthMediatorPropertiesEditionPartForm.java

```diff
@@ -123,13 +126,12 @@ public Composite createFigure(final Composite parent, final FormToolkit widgetFa
 	public void createControls(final FormToolkit widgetFactory, Composite view) {
 		CompositionSequence oAuthMediatorStep = new BindingCompositionSequence(propertiesEditionComponent);
 		CompositionStep propertiesStep = oAuthMediatorStep.addStep(EsbViewsRepository.OAuthMediator.Properties.class);
-		propertiesStep.addStep(EsbViewsRepository.OAuthMediator.Properties.description);
 		propertiesStep.addStep(EsbViewsRepository.OAuthMediator.Properties.commentsList);
 		propertiesStep.addStep(EsbViewsRepository.OAuthMediator.Properties.reverse);
 		propertiesStep.addStep(EsbViewsRepository.OAuthMediator.Properties.remoteServiceUrl);
 		propertiesStep.addStep(EsbViewsRepository.OAuthMediator.Properties.username);
 		propertiesStep.addStep(EsbViewsRepository.OAuthMediator.Properties.password);
-		
+        propertiesStep.addStep(EsbViewsRepository.OAuthMediator.Properties.description);
```

Comment:
Correct formatting.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/806